test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -741,7 +742,7 @@ public void TensorFlowTransformCifar()
             IDataView trans = new TensorFlowTransformer(mlContext, tensorFlowModel, "Input", "Output").Transform(pixels);
 
             trans.Schema.TryGetColumnIndex("Output", out int output);
-            using (var cursor = trans.GetRowCursor(col => col == output))
+            using (var cursor = trans.GetRowCursorForAllColumns())
```

Comment:
>GetRowCursorForAllColumns [](start = 38, length = 25)

GetRowCursor(trans.Schema["Output"]) and you can kill output variable above.

Reply:
Uh.. thanks. 

---
In reply to: [249123646](https://github.com/dotnet/machinelearning/pull/1975#discussion_r249123646) [](ancestors = 249123646)